src/reqstool/model_generators/requirements_model_generator.py

```diff
@@ -289,15 +290,16 @@ def __parse_requirements(self, data):  # NOSONAR
                 # Check if rationale is defined, or set it to None
                 rationale = req["rationale"] if "rationale" in req else None
 
+                urn_id = UrnId(urn=urn, id=req["id"])
                 req_data = RequirementData(
-                    id=UrnId(urn=urn, id=req["id"]),
+                    id=urn_id,
                     title=req["title"],
                     significance=SIGNIFANCETYPES(req["significance"]),
                     description=req["description"],
                     rationale=rationale,
-                    category=req["category"],
+                    categories=req["categories"],
```

Comment:
Need to validate input using the relevant enum. You can probably use list comprehension similar to:

`categories = [CATEGORYTYPES[category] for  category in req["categories"]]`

Reply:
Fixed in latest commit